docs/fast_data/inputs_and_outputs.md

```diff
@@ -4,24 +4,26 @@ title: Inputs and Outputs
 sidebar_label: Inputs and Outputs
 ---
 
-This page presents all the different inputs and outputs that revolve around the Fast Data ecosystem, including both the core user data in Projections and Single Views, and the events concerning errors, successful data generation, and so on.
+The fast-data has an Event Driven Architecture, as such there are many events that make everything work. In this page we will explore these events in detail to understand better the whole fast-data lifecycle.
```

Comment:
Better call it _Fast Data_ instead of _fast-data_. I'd suggest this:


```suggestion
The Fast Data ecosystems has an Event Driven Architecture, as such there are many events that make everything work. In this page we will explore these events in detail to understand better the whole fast-data lifecycle.
```